src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesViewModel.cs

```diff
@@ -21,11 +21,15 @@ public class ConfigurationFilesViewModel
         public ObservableCollection<string> ConfigurationFilesCollection { get; private set; }
         public string? SelectedPath { get; set; }
         public ICommand OpenConfigurationFile { get; }
+        private ISettings _settings;
+        private INuGetProjectContext _projectContext;
 
-        public ConfigurationFilesViewModel()
+        public ConfigurationFilesViewModel(ISettings settings, INuGetProjectContext projectContext)
         {
             ConfigurationFilesCollection = new ObservableCollection<string>();
             OpenConfigurationFile = new DelegateCommand(ExecuteOpenConfigurationFile, IsSelectedPath, NuGetUIThreadHelper.JoinableTaskFactory);
+            _settings = settings;
+            _projectContext = projectContext;
```

Comment:
Enforce the scenario where the params are null

```suggestion
            _settings = settings ?? throw new ArgumentNullException(nameof(settings));
            _projectContext = projectContext  ?? throw new ArgumentNullException(nameof(projectContext));
```